dynamiq/nodes/agents/base.py

```diff
@@ -104,12 +107,15 @@ def _init_prompt_blocks(self):
             "tools": "{tool_description}",
             "instructions": "",
             "output_format": "Provide your answer in a clear and concise manner.",
+            "relevant_information": "{relevant_memory}",
+            "conversation_history": "{context}",
```

Comment:
I am curious if the order of blocks matters. Maybe it's better to first have blocks with relevant information and conversation history, and then add the block with the output format closer to the end so that the LLM remembers it better.

Also, do you think "relevant information" and "relevant_memory" are the same? I would prefer to rename the block to "relevant information from previous conversation."

Additionally, it might be helpful to mention somewhere in the prompt that the chat conversation is for context, so it should only be used if it's helpful for answering, etc.